docs/jwst.rst

```diff
@@ -223,14 +217,31 @@ SI WFE
 SI internal WFE measurements are from ISIM CV3 testing (See JWST-RPT-032131 by David Aronstein et al.)
 The SI internal WFE measurements are distinct for each of the modules and
 channels. When enabled, these are added to the final pupil of the optical
-train, i.e. after the coronagraphic image planes.
-
-**Caution** WFE is not yet included for the coronagraph field points; it will use the closest imaging field point. See issue #180.
+train, i.e. after the coronagraphic image planes. For field-points outside of
+the measurement bounds, WebbPSF performs an extraction routine.
```

Comment:
should this be 'extrapolation' rather than extraction?

Reply:
Yup. Midnight typos.